src/main/java/uk/nhs/digital/nhsconnect/lab/results/model/edifact/ReportStatusCode.java

```diff
@@ -21,4 +26,7 @@ public static ReportStatusCode fromCode(@NonNull String code) {
             .orElseThrow(() -> new IllegalArgumentException("No Report Status Code for '" + code + "'"));
     }
 
+    public static ProcedureRequest.ProcedureRequestStatus mapToProcedureRequestStatus(ReportStatusCode code) {
```

Comment:
Could this be moved to the `ProcedureRequestMapper` class?

Personally, I'm not a fan of it being here, because I think the responsibility for mapping the Report Status Code from an `Edifact` string to a FHIR `ProcedureRequestStatus` code should be within the mapper.

Reply:
Is it better now? 818a10323f2cbd4f2ab584b1908df2510e7a4d14 Thanks!